osmosis/assetlist.json

```diff
@@ -165,6 +165,50 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdc.svg"
       }
     },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/0B3C3D06228578334B66B57FBFBA4033216CEB8119B27ACDEE18D92DA5B28D43",
+          "exponent": 0,
+          "aliases": [
+            "avalanche-uusdc"
+          ]
+        },
+        {
+          "denom": "avalanche-usdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/0B3C3D06228578334B66B57FBFBA4033216CEB8119B27ACDEE18D92DA5B28D43",
+      "name": "Wormhole USDC(Avalanche)",
+      "display": "avalanche-usdc.hole",
+      "symbol": "avaxUsdc.hole",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "avalanche",
+            "base_denom": "0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E"
+          },
+          "provider": "Wormhole"
```

Comment:
This is good but not quite right. There is a no direct Wormhole bridge from Avalanche to Osmosis. We'll need to register this avalanche.USDC asset under the /gateway/ chain, defining this Wormhole bridge in the traces there. And then we may define the asset (avalanche.USDC.wh) here under /osmosis/, and the trace on this one would point to gateway::avalanche.USDC.

Reply:
alright, let me attempt that.

Reply:
done, how does it look like now? I am only dubious about the images, can you check that up, in both the gateway and osmosis side? thanks!